pages/configuring_ide_netbeans.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Configuring your IDE
+permalink: /configuring-ide-netbeans/
+redirect_from:
+  - /configuring_ide.html
```

Comment:
I'll add it for consistency... I'm going to merge this, don't worry that's my job now :-)